src/mail.sh

```diff
@@ -774,6 +815,13 @@ function parse_mail_options()
   if [[ "$setup_token" == 1 ]]; then
     validate_setup_opt
   fi
+
+  if [[ "$commit_ref" == 0 ]]; then
+    if ! check_revision "${options_values['PASSTHROUGH']}"; then
+      # assume last commit if none given
+      options_values['PASSTHROUGH']="$(str_strip "${options_values['PASSTHROUGH']} @^")"
```

Comment:
I forgot the meaning of `@^`; could you add a comment about that in the code?

Reply:
Sure, `@` is just shorthand for `HEAD`.